incubator/kube2iam/templates/daemonset.yaml

```diff
@@ -0,0 +1,51 @@
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  name: {{ template "fullname" . }}
+  labels:
+    app: {{ template "fullname" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    heritage: "{{ .Release.Service }}"
+    release: "{{ .Release.Name }}"
+spec:
+  selector:
+    matchLabels:
+      app: {{ template "fullname" . }}
+  template:
+    metadata:
+      labels:
+        app: {{ template "fullname" . }}
+        release: "{{ .Release.Name }}"
+    spec:
+      hostNetwork: true
+      containers:
+      - name: kube2iam
+        image: "{{ .Values.image }}:{{ .Values.imageTag }}"
+        imagePullPolicy: {{ default "" .Values.imagePullPolicy | quote }}
+        args:
+        {{- if .Values.host.iptables }}
+        - --host-ip={{ default "$(HOST_IP)" .Values.host.ip }}
+        {{- end }}
+        - --iptables={{ default false .Values.host.iptables }}
+        {{- if .Values.host.interface }}
+        - --host-interface={{ default "docker0" .Values.host.interface | quote }}
+        {{- end }}
+        {{- if .Values.containerPort }}
```

Comment:
Same as above. No `if` needed because kube2iam listens on `8181` by default. No risk in passing the argument regardless of whether `.Values.containerPort` is defined.

This one can move up in the args list, too. Again, IMHO.